### PR TITLE
Added the "-l" flag to download the latest episode

### PR DESCRIPTION
**Pull request by asbjorn**
I think this will fix the issue #36.
This new feature identifies the latest available episode for the last season available.

Tested for the NRK series:
    - ./nrk-tv-downloader.sh -l https://tv.nrk.no/serie/foerstegangstjenesten
    - ./nrk-tv-downloader.sh -l https://tv.nrk.no/serie/113

Tested on my Ubuntu 20.04 LTS.

### CODE REVIEW

**Comment by asbjorn** [comment]
Could you add the "hacktoberfest-accepted" label **or** add the topic "hacktoberfest" on the project?

**Comment by odinuge** [comment]
Awesome! Will take a proper look tomorrow!

> Could you add the "hacktoberfest-accepted" label or add the topic "hacktoberfest" on the project?

Yeah, did that now! 😃

**Review by odinuge — COMMENTED** [review]
Awesome!

Just a few comments, but other than that it looks good.

Also, would be nice if you could run `shfmt -w nrk-tv-downloader.sh` to format the code properly. :smile:

File: nrk-tv-downloader.sh [hunk 1]
```diff
@@ -584,8 +585,56 @@ function program() {
...
+        "https://psapi.nrk.no/tv/catalog/series/${series_id}")
+    local last_season=$(echo "$series_data" | jq '._links.seasons[-1]')
+    local last_season_nr=$(echo "$last_season" | jq -r '.name')
+    if [ -z "$last_season_nr" ]; then
+        echo "Unable to find season for series: ${series_id}"
+        return
+    fi
+    local last_season_href=$(echo "$last_season" | jq -r '.href')
+
+    local url2="https://psapi-we.nrk.no${last_season_href}"
```
[the remark this answers is not in the document]
Comment (asbjorn) [reply nrk-tv-downloader.sh:622]: You're right - I'll fix that asap.

**Review by odinuge — APPROVED** [review]
ltgm

Thanks again!

**Pull request merged**